src/uu/tail/src/tail.rs

```diff
@@ -214,39 +243,51 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
     } else {
         let multiple = files.len() > 1;
         let mut first_header = true;
-        let mut readers = Vec::new();
+        let mut file_objs = Vec::new();
```

Comment:
file_objs ? I would prefer you use an explicit variable name. 
Not variable name explaining the type